src/runtime_src/core/pcie/tools/cloud-daemon/pciefunc.cpp

```diff
@@ -84,7 +84,7 @@ bool pcieFunc::loadConf()
     std::vector<std::string> config;
     std::string err;
 
-    dev->sysfs_get("", "config_mailbox_channel_switch", err, chanSwitch);
+    dev->sysfs_get("", "config_mailbox_channel_switch", err, chanSwitch, -1);
```

Comment:
default should be 0 here, which means that all mailbox msgs should go through HW mailbox.